hibernate-core/src/main/java/org/hibernate/jpa/QueryHints.java

```diff
@@ -91,30 +90,27 @@
 	 * 
 	 * Note: Currently, attributes that are not specified are treated as FetchType.LAZY or FetchType.EAGER depending
 	 * on the attribute's definition in metadata, rather than forcing FetchType.LAZY.
-	 *
-	 * @deprecated (since 5.4) Use {@link GraphSemantic#FETCH}'s {@link GraphSemantic#getJpaHintName()} instead
```

Comment:
Are the "deprecated" removals intentional?

Reply:
Yes.